winrm/client.go

```diff
@@ -2,19 +2,25 @@ package winrm
 
 import (
 	"bytes"
-	"crypto/tls"
 	"crypto/x509"
 	"fmt"
 	"io"
-	"net/http"
+	"launchpad.net/gwacl/fork/http"
```

Comment:
Do we really need to use this fork?
What's the compelling reasons in using this version compared to the original golang version?
I'm very very reluctant using something that is not part of the golang stdlib for such low-level library, especially for security reasons.